sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -23,11 +23,11 @@ static az_span const AZ_KEYVAULT_KEY_TYPE_KEY_STR = AZ_CONST_STR("keys");
 // static az_span const AZ_KEYVAULT_KEY_TYPE_SECRET_STR = AZ_CONST_STR("secrets");
 // static az_span const AZ_KEYVAULT_KEY_TYPE_CERTIFICATE_STR = AZ_CONST_STR("certificates");
 
-static az_span const AZ_KEYVAULT_WEB_KEY_TYPE_EC_STR = AZ_CONST_STR("EC");
-static az_span const AZ_KEYVAULT_WEB_KEY_TYPE_EC_HSM_STR = AZ_CONST_STR("EC-HSM");
-static az_span const AZ_KEYVAULT_WEB_KEY_TYPE_RSA_STR = AZ_CONST_STR("RSA");
-static az_span const AZ_KEYVAULT_WEB_KEY_TYPE_RSA_HSM_STR = AZ_CONST_STR("RSA-HSM");
-static az_span const AZ_KEYVAULT_WEB_KEY_TYPE_OCT_STR = AZ_CONST_STR("oct");
+az_span const AZ_KEYVAULT_WEB_KEY_TYPE_EC_STR = AZ_CONST_STR("EC");
+az_span const AZ_KEYVAULT_WEB_KEY_TYPE_EC_HSM_STR = AZ_CONST_STR("EC-HSM");
+az_span const AZ_KEYVAULT_WEB_KEY_TYPE_RSA_STR = AZ_CONST_STR("RSA");
+az_span const AZ_KEYVAULT_WEB_KEY_TYPE_RSA_HSM_STR = AZ_CONST_STR("RSA-HSM");
+az_span const AZ_KEYVAULT_WEB_KEY_TYPE_OCT_STR = AZ_CONST_STR("oct");
```

Comment:
Theses are not static but the lines below are static. Is there a reason for this? We should be consistent unless there is a good reason.

Reply:
static const variables need to be used somewhere in the code or we get a compilation warning. 
We use them for constants we use internally like the ones below, where `keys` and `create` are used by client to build url path.

non-static const variables are defined in a header file as `external const` and assigned here. We use them to expose constants to customer. Like in this case, we are exposing all this spans to customers so they can use them. Compiler won't complain if they are not used anywhere

Reply:
Could we just use inline functions and forget about static/extern variables which are quite painful to use and to understand (for example initialization order for dependent objects)? Also, static variables can't be optimized but inline functions can.